_posts/2016/2016-06-14-gradle-jenkins-plugin.md

```diff
@@ -82,13 +83,13 @@ This will provide you the list of available tasks:
 {% highlight bash %}
 ./gradlew validateJenkinsItems
 {% endhighlight %}
-This will validate local changes with your Jenkins
+This will validate local changes with your Jenkins.
 
 -----------------------
 {% highlight bash %}
 ./gradlew updateJenkinsItems
 {% endhighlight %}
-This will update your Jenkins installation with the dsl defined at your jenkins.gradle
+This will update your Jenkins installation with the DSL defined localy.
```

Comment:
This will update your Jenkins installation with the DSL defined localy.
>
This will update your Jenkins installation with the DSL defined locally.